### ISSUE TITLE
[Programmatic Access - Make code for micro bit - Feedback]: Alternate text is not defined for the image present inside the “Usabilla Feedback button”

### ISSUE THREAD

**Issue by Chandangithb** [microsoft/pxt-microbit#4839]
### User Experience:
Visually impaired users who depend on screen reader and navigate in scan mode will not be able to understand the information provided or purpose of the image properly and might get confused.

Note: User credentials should NOT be included in the bug.

**### Repro Steps:**

1. Launch URL: [https://arcade.makecode.com/--skillmap#docs:/skillmap/racer ](https://arcade.makecode.com/--skillmap#docs:/skillmap/racer)
2. "Monster Racer" sign in page appears.
3. Navigate to the "Feedback" button and run an accessibility insights tool.
4. Observe and verify the issue.

### Actual Result:
Issue: Alternate text is not defined for the image present inside the “Usabilla Feedback button” as a result when focus moves to the image in scan mode (Narrator) announces as “To get the missing image description open context menu unlabeled graphic”.

### Expected Result:
Solution: Alternate text should be defined for the image present inside the “Usabilla Feedback button” as “Feedback logo”.

### Issue: 
Ensures <img> elements have alternate text or a role of none or presentation (image-alt - [](https://accessibilityinsights.io/info-examples/web/image-alt))

### Target application: 
MakeCode Skill Map - https://arcade.makecode.com/--skillmap#docs:/skillmap/racer

### Element path: 
#usabilla_live_button_container_iframe898772216;img

### Snippet: 
`<img src="//d6tizftlrpuof.cloudfront.net/live/resources/buttons/usabilla_orange_rightSideFeedback.png" width="39" height="130">`

### How to fix:
Fix any of the following:
 Element does not have an alt attribute.
 aria-label attribute does not exist or is empty.
 aria-labelled by attribute does not exist, references elements that do not exist or references elements that are empty.
 Element has no title attribute.
 Element's default semantics were not overridden with role="none" or role="presentation".

### Environment: 
Microsoft Edge version 105.0.1300.0

This accessibility issue was found using Accessibility Insights for Web 2.32.0 (axe-core 4.4.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at [](http://aka.ms/AccessibilityInsights).

### MAS Reference:

- https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7b20E8F5D5-3053-4A0D-AD22-A35207B8F892%7d&file=MAS%201.1.1%20%E2%80%93%20Non-text%20Content.docx&action=default&mobileredirect=true

 ### Reference Links:
- [Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
- External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [External Bug Process (sharepoint.com)](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDE1MThmM2JlZGFjMDRlZDhiZmViYTUxMDA2M2NiZWI3fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3NDEyNTM2MTEzNzIwNzR8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaUlpd2lRVTRpT2lJaUxDSlhWQ0k2TVRKOQ%3D%3D&sdata=MnlpSXlVMnpGT3cvZ0xJNnFWUWZWTGt6dmo1VkJMZFpIZHBvUUduNDhyMD0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-kgeetala%40microsoft.com). “HCL Staff should not log any third-party external bugs. Should be routed to EDAD team”.
- Please reach out to [C&AI Teams channel](https://apc01.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906844600&sdata=22JjBMoA1ID/KnYqFa/OB%2BrkTaLh5YYsGBd/7Bm7u5w%3D&reserved=0) for any process-related queries.

**Comment by Chandangithb** [comment]
#A11yAuto, #K4W, #A11yMAS, #A11ySev3, #HCL, #Accessibility, #WCAG1.1.1, #HCL-MakeCode-New

**Comment by abchatra** [comment]
This is filed on incorrect repo. Moving to pxt-arcade repo.
